modules/50_damages/KWTCint/datainput.gms

```diff
@@ -0,0 +1,57 @@
+*** |  (C) 2006-2020 Potsdam Institute for Climate Impact Research (PIK)
+*** |  authors, and contributors see CITATION.cff file. This file is part
+*** |  of REMIND and licensed under AGPL-3.0-or-later. Under Section 7 of
+*** |  AGPL-3.0, you are granted additional permissions described in the
+*** |  REMIND License Exception, version 1.0 (see LICENSE file).
+*** |  Contact: remind@pik-potsdam.de
+*** SOF ./modules/50_damages/KWTCint/datainput.gms
+
+* satisfy dependencies
+$ifi not %downscaleTemperature% == 'CMIP5' abort "module damages=KWTCint requires downscaleTemperature=CMIP5";
+
+** damage specification
+    
+
+*default specification:
```

Comment:
a comment where these values come from would be appreciated

Reply:
done